src/GoTrueClient.ts

```diff
@@ -593,6 +595,43 @@ export default class GoTrueClient {
     }
   }
 
+  /**
+   * Returns a new session, from the current session, regardless of expiry status.
+   * If the access token or refresh token are invalid, an error will be thrown.
+   * @param currentSession The current session that minimally contains an access token and refresh token.
+   */
+  async refreshSession(
+    currentSession: Pick<Session, 'access_token' | 'refresh_token'>
+  ): Promise<AuthResponse> {
+    try {
+      if (!currentSession.access_token || !currentSession.refresh_token) {
+        throw new AuthSessionMissingError()
+      }
+
+      const payload = decodeJWTPayload(currentSession.access_token)
+      if (!payload.exp) {
+        throw new CustomAuthError('Invalid access token!', 'AuthTokenInvalidError', 400)
```

Comment:
Can you please rename this error differently. I'm fine with using `AuthInvalidJWTError` or something of that sort. This also does not have a status code, since it does not go out to the network.

Reply:
Since the `access_token` isn't need to refresh, then I'll remove this code. You want me to go ahead and remove the `decodeJWTPayload()` helper as well, and let your PR take care of it?

Reply:
I see your PR was already merged. I'll remove the decode function from my branch and sync changes.